### PR TITLE
add: how orient text to center, right and how to quoting text

### PR DESCRIPTION
**Pull request by donaldte**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,26 @@ print("Hi there")
...
+
+
+
+<div align="right"> Orient text to the right Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</div>
+
+## Quoting text
```
Comment (zairahira) [inline README.md]: IMO, a better title would be:

```suggestion
## Nesting Quotes
```

File: README.md [hunk 2]
```diff
@@ -105,6 +105,26 @@ print("Hi there")
...
+
+
+
+<div align="right"> Orient text to the right Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</div>
```
Comment (zairahira) [inline README.md]: Thank you for suggesting these changes @donaldte. Text orientation is more specific to HTML rather than Markdown. 

```suggestion
```

**Review by zairahira — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (donaldte) [reply README.md]: Alright @zairahira  I get it thanks

**Review by zairahira — APPROVED** [review]
Thank you for contributing @donaldte 🎉

**Pull request merged**